### DISCUSSION TITLE
GBL model works fine, but it has no "texture?".

### DISCUSSION THREAD

**Discussion by LoSilopante** [mozilla/hubs#3241] [Avatar Questions] [answered]
I made a .GBL in paint 3D and inside avatar selection it's shown as having colors and everything but inside an hub it becomes transparent.
![Screenshot (125)](https://user-images.githubusercontent.com/24480520/97419623-9085fe80-190a-11eb-8758-6b764f598b54.png)

**Comment by j-conrad** [comment]
I can confirm I'm seeing similar behavior.

The model I'm using also came from Paint3D. 
The one on the left was published from Spoke while the one on the right was dropped straight into Hubs.
Looks like something isn't being handled properly, most likely the fact that Paint3D models are using a Specular/Glossiness workflow instead of Metallic/Roughness.
![image](https://user-images.githubusercontent.com/31968496/97610762-8a5b5500-19d2-11eb-8899-d48d6b884a9e.png)


Are you also publishing from Spoke or are you putting the model straight into Hubs?

**Comment by robinkwilson** [comment, marked as answer]
Created an issue to follow: https://github.com/mozilla/hubs/issues/3299
